### PR TITLE
docs: clarify the access level granted by the "home" interface

### PR DESCRIPTION
**Pull request by mvo5**
Closes:#1570621

### CODE REVIEW

File: docs/interfaces.md [hunk 1]
```diff
@@ -54,8 +54,9 @@ Auto-Connect: yes
 
 ### home
 
-Can access non-hidden files in user's $HOME. This is restricted
-because it gives file access to all of the user's $HOME.
+Can access non-hidden files in user's $HOME to read/write/lock.
+This is restricted because it gives file access to all of the user's
+$HOME.
```
Comment (niemeyer) [inline docs/interfaces.md]: Is that actually true?  I think it forbids access to hidden files, right?

Comment (niemeyer) [inline docs/interfaces.md]: Nevermind, sorry.. it still says so right above.

Comment (niemeyer) [inline docs/interfaces.md]: UPDATE: Can we please `s/all of//`. Second sentence contradicts the first.

**Comment by chipaca** [comment]
Also, backticks around `$HOME` please. Other than that, and Gustavo's point, :+1:

**Comment by mvo5** [comment]
Thanks, I addressed the feedback.

**Comment by pedronis** [comment]
@mvo5 missing backsticks around the first $HOME ?

**Comment by mvo5** [comment]
@pedronis silly me, thanks!

**Comment by zyga** [comment]
+1

**Pull request merged**